index.html

```diff
@@ -19633,6 +19677,29 @@ <h2 id="AudioWorklet-Examples">
           </li>
         </ul>
       </section>
+      <section>
+        <h2>
+          Unloading a document
+        </h2>
+        <p>
+          Additional <a href=
+          "https://html.spec.whatwg.org/#unloading-document-cleanup-steps">unloading
+          document cleanup steps</a> are defined for document that uses
+          <a>BaseAudioContext</a>:
+        </p>
+        <ul>
+          <li>Reject all the promises of <a>pendingPromises</a> with
+          <code>InvalidStateError</code>, for each <a>AudioContext</a> and <a>
+            OfflineAudioContext</a> of this document.
```

Comment:
"of this document" is kind of imprecise.

I think you'll want: for each AudioContext and OfflineAudioContext whose relevant global object is the same as the document's associated Window.